db.js

```diff
@@ -0,0 +1,54 @@
+// const mongoose = require('mongoose');
+
+const mongodb = require('mongodb');
+const MongoClient = mongodb.MongoClient;
+
+//Set up default mongoose connection
+var mongoDBLocal = 'mongodb://localhost:27017'
```

Comment:
The similar work needs to be done here. 
This will work find on development and it will connect to the dev db. (localhost)

But when it's deployed on the production, we won't work there because it doesn't have a dev db on a production machine. 

Let's put a condition like we had done before on the client. https://github.com/nishi-ai/softball-register-web/blob/d5c8c2eb2328cb47a9f463244ac8a478a46787d0/src/pages/NewPlayerPage.js#L5

We can use `process.env.NODE_ENV` to see if it's running on a production or a dev.